### ISSUE TITLE
Embed color change

### ISSUE THREAD

**Issue by Jay-Madden** [patrickklaeren/Accord#29]
Currently the black embed color in my opinion looks very bad. I propose changing the embed color to the same shade of green as the icon, that would make it fit together well

**Comment by jeuxjeux20** [comment]
I agree that the embed color does *not* LGTM. The primary color of Accord's design is green, so it's indeed a great option.

**Comment by patrickklaeren** [comment]
If I'm honest this is a bit too broad to action right now. This probably needs to be on a case-by-case basis.
